cmd/zdb/zdb.c

```diff
@@ -27,6 +27,10 @@
  * Copyright (c) 2017, 2018 Lawrence Livermore National Security, LLC.
  * Copyright (c) 2015, 2017, Intel Corporation.
  * Copyright (c) 2020 Datto Inc.
+ * Copyright (c) 2020 The FreeBSD Foundation
+ *
+ * Portions of this software were developed by Allan Jude
+ * under sponsorship from the FreeBSD Foundation.
```

Comment:
You sure legal copyright ownership lays with the FreeBSD foundation for your work? You have a transfer of ownership contract (as part of your sponsorship)?
If not, you are personal the legal copyright holder, not the foundation.

I doubt the headers are meant for everyone to (byond copyright) add they are the one working for their boss in an added note though.

Reply:
The copyright is assigned as part of the contract, this specific text is also a requirement of the contract.

Reply:
@allanjude Loud and clear, thanks for clearifying.

Reply:
If there aren't any arguments against, I'd update this to " * Copyright (c) 2020 The FreeBSD Foundation. All rights reserved." to keep it consistent

Reply:
Thats legally no issue at all, as discussed before... :)

Reply:
The FreeBSD Project, and FreeBSD Foundation specifically do not use the text 'all rights reserved' because it is nor meaningful anymore

see: https://svnweb.freebsd.org/base?view=revision&revision=333391

Reply:
Oh, interesting! Would you suggest to adapt the others, too, then?

Reply:
> 
> 
> Oh, interesting! Would you suggest to adapt the others, too, then?

We cannot change the copyright messages owned by others, but you can update your own to remove the problematic clause.

Reply:
@allanJude the discussion for that has been closed and you are spreading misinformation

Reply:
Copyright headers are barely "meaningfull" to begin with, "All rights reserved" doesn't have a meaning, either leaving it or removing it is irrelevant and perfectly fine.